vendor/github.com/coreos/go-systemd/activation/listeners.go

```diff
@@ -48,8 +48,6 @@ func TLSListeners(unsetEnv bool, tlsConfig *tls.Config) ([]net.Listener, error)
 	}
 
 	if tlsConfig != nil && err == nil {
-		tlsConfig.NextProtos = []string{"http/1.1"}
```

Comment:
What I did was just updating vendors with glide.

More precisely, this line in go-systemd was removed by https://github.com/coreos/go-systemd/commit/d736403b010a1169d7a34a9a638b26f07a0658b1.
The change was supposed to be included in https://github.com/coreos/fleet/pull/1671, but actually for some reason it was not included there. That's why you're seeing this in this PR. I think the change should stay there, because glide.lock is already pointing to version https://github.com/coreos/go-systemd/commit/422fafb691792ca9fdf7097a5ebd49f10839676c of `go-systemd` that contains the https://github.com/coreos/go-systemd/commit/d736403b010a1169d7a34a9a638b26f07a0658b1.